### PR TITLE
update @angular/core@11 @angular/cli@11

### PR DESCRIPTION
**Pull request by yasinkocak**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -33,18 +33,17 @@
...
+    "@angular/language-service": "^11.2.14",
+    "@types/jasmine": "~3.6.0",
     "@types/jasminewd2": "~2.0.6",
     "@types/node": "^15.12.2",
-    "codelyzer": "^6.0.2",
+    "codelyzer": "^6.0.0",
     "coveralls": "^3.0.3",
-    "jasmine-core": "^3.7.1",
-    "jasmine-spec-reporter": "^7.0.0",
+    "jasmine-core": "~3.6.0",
```
Comment (emielb) [inline package.json:45]: Why have you downgraded the jasmine packages? If not for any particular reason, please revert.

Comment (yasinkocak) [reply package.json:45]: Angular cli migration take care of this automatically

**Comment by emielb** [comment]
Fixes #169

File: package.json (hunk 1 above)
Comment (emielb) [reply package.json:45]: Will be done after the angular 12 upgrade

**Review by emielb — APPROVED** [review]

**Pull request merged**